tests/test_manipulation.py

```diff
@@ -665,6 +667,71 @@ def test_minimum_signed_integers(self, data, dtype):
         assert_array_equal(result, expected)
 
 
+class TestRequire:
+    flag_names = ["C", "C_CONTIGUOUS", "F", "F_CONTIGUOUS", "W"]
```

Comment:
Should we extend with `CONTIGUOUS` and `FORTRAN`?

Reply:
see this [comment](https://github.com/IntelPython/dpnp/pull/2036#discussion_r1759784189)